CLOUD-1529: Reorganization of Cloud Edition section (#1371)
akeneo/pim-docs: cloud_edition/flexibility_mode/docs/job_consumers_and_workers.rst

be executed in a timely manner, and that their execution will not interfere with other business
operations.

Configurations of daemons are stored under `/home/akeneo/.systemd/pim_job_queue`.

Each text file contains the list of supported jobs for a given queue.

Example in file `/home/akeneo/.systemd/pim_job_queue/1.conf`:

.. code-block:: bash
   :linenos:

   csv_product_export
   csv_category_export
   csv_family_export

The name of the file is the **daemon identifier**. Make it simple and use only integer numbers.

.. code-block:: bash
   :linenos:

   # Launch the daemon for this configuration /home/akeneo/.systemd/pim_job_queue/1.conf
   partners_systemctl pim_job_queue@1 start

   # Check the status of the daemon #2
   partners_systemctl pim_job_queue@2 status

    # Check the status of all daemons
   partners_systemctl pim_job_queue@* status

   # See real time logs for daemon #3
   journalctl --unit=pim_job_queue@3 -f

Please note that if **no configuration** file exist for a given daemon identifier,
the daemon will consider that it has to consume **any** elements in the queue. This is the default
behavior.

Onboarder
---------

> Onboarder requires workers to run at all times, it is disabled by default as not everybody needs it.

Reviewer: ```suggestion
While Onboarder requires workers to run at all times, those are disabled by default since some customers do not use the Onboarder.
```
